src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -68,19 +124,72 @@
 {# Print the result #}
 
 <section class="{{ _css_class }}"{{ _extra_attributes|raw }}>
-{% if _picture.img is not empty and _variant != 'plain-background'  %}
-  <figure class="ecl-banner__picture-container">
-    {% include '@ecl/picture/picture.html.twig' with {
-      picture: _picture,
-      extra_classes: 'ecl-banner__picture',
-      extra_image_classes: 'ecl-banner__image',
-      extra_attributes: [{ name: 'data-ecl-banner-image' }],
-    } only %}
-  {% if _credit is not empty and _variant != 'plain-background' %}
-    <footer class="ecl-banner__credit"><small>{{ _credit }}</small></footer>
+  {# Media #}
+  {% if _picture.img is not empty %}
+    {# Image #}
+    <figure class="ecl-banner__picture-container">
+      {% include '@ecl/picture/picture.html.twig' with {
+        picture: _picture,
+        extra_classes: 'ecl-banner__picture',
+        extra_image_classes: 'ecl-banner__image',
+        extra_attributes: [{ name: 'data-ecl-banner-image' }],
+      } only %}
+    {% if _credit is not empty %}
+      <footer class="ecl-banner__credit"><small>{{ _credit }}</small></footer>
+    {% endif %}
+    </figure>
+  {% elseif _video is not empty %}
+    {# Video #}
+    {% set _video_sources = _video.sources|default({}) %}
+    {% set _video_tracks = _video.tracks|default({}) %}
```

Comment:
i think it would make sense to do the same we do for the picture also for the video now that we are spreading it in different components. Maybe not in this pr, but that way we could also add support for custom classes and attributes.